### PR TITLE
2424: increase filter button size. Make labels multiline

### PR DESCRIPTION
**Pull request by f1sh1918**
### Short description

-Increase button size to allow multiple lines for labels

### Resolved issues

#2424

### CODE REVIEW

File: web/src/components/base/ToggleButton.tsx [hunk 1]
```diff
@@ -11,13 +11,18 @@ const StyledButton = styled.button<{ $active: boolean | null }>`
...
+  height: 100px;
   background-color: ${props => (props.$active ? props.theme.colors.themeColor : props.theme.colors.backgroundColor)};
   color: ${props => props.theme.colors.textSecondaryColor};
   padding: 8px;
   cursor: pointer;
 `
 
+const Text = styled(StyledSmallViewTip)`
+  white-space: normal;
+  word-break: break-word;
```
Comment (SteffiStuffel) [inline web/src/components/base/ToggleButton.tsx:23]: Why have you put this here and not inside StyledSmallViewTip? Wouldn't other usages of `StyledSmallViewTip` have the same problem?

**Review by SteffiStuffel — APPROVED** [review]
Tested with obdach

**Review by LeandraH — APPROVED** [review]
Looks good, tested in Firefox

File: web/src/components/base/ToggleButton.tsx (hunk 1 above)
Comment (f1sh1918) [reply web/src/components/base/ToggleButton.tsx]: good find. changed this in last commit

**Pull request merged**